<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_1"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="msgButton">
       <property name="text">
        <string>Submit Message to Experiment State Tracker</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="update_reminder_timer_label">
       <property name="text">
        <string>Update Reminder Timer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="time_remaining_label">
         <property name="text">
          <string>Remaining (mm:ss):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="time_remaining_rbv">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>6</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>15:00</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="timer_setting_label">
         <property name="text">
          <string>Setting (mins):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="PyDMPushButton" name="PyDMPushButton">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>Reset Timer</string>
         </property>
         <property name="channel" stdset="0">
          <string>ca://IOC:${endstation}:EXPSTATE:IncUpdate.PROC</string>
         </property>
         <property name="pressValue" stdset="0">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="timer_setting_edit">
         <property name="text">
          <string>15</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="PyDMDrawingArc" name="clock_arc">
          <property name="geometry">
           <rect>
            <x>15</x>
            <y>15</y>
            <width>130</width>
            <height>130</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>130</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>130</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="penWidth" stdset="0">
           <double>5.000000000000000</double>
          </property>
          <property name="rotation" stdset="0">
           <double>0.000000000000000</double>
          </property>
          <property name="startAngle" stdset="0">
           <double>90.000000000000000</double>
          </property>
          <property name="spanAngle" stdset="0">
           <double>245.000000000000000</double>
          </property>
         </widget>
         <widget class="PyDMDrawingCircle" name="alarm_circle">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMDrawingArc</class>
   <extends>QWidget</extends>
   <header>pydm.widgets.drawing</header>
  </customwidget>
  <customwidget>
   <class>PyDMDrawingCircle</class>
   <extends>QWidget</extends>
   <header>pydm.widgets.drawing</header>
  </customwidget>
  <customwidget>
   <class>PyDMPushButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.pushbutton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
